### PR TITLE
D8 support

### PR DESCRIPTION
**Pull request by finalhow**
This branch detects if the path is a Composer installed Drupal 8 site, or not.  If it is, then it will add and run slightly modified permission changes on the parent path and the parent/vendor folder.  It will also run restorecon on the parent path.

Please verify and merge in when ready.

### CODE REVIEW

File: templates/drupal-fix-permissions.sh.erb [hunk 1]
```diff
@@ -72,7 +74,30 @@ if [ -z "${drupal_user}" ] || [[ $(id -un "${drupal_user}" 2> /dev/null) != "${d
   exit 1
 fi
 
-cd $drupal_path
+printf "Checking Drupal version at "${drupal_path}"... ";
+drupal_version=$(drush status --root=${drupal_path} | grep 'Drupal version' | awk -F: '{print $2}' | awk -F. '{print $1}' | tr -d [:space:])
```
Comment (minorOffense) [inline templates/drupal-fix-permissions.sh.erb]: I'd just use ``drush status --field=drupal-version``

Give it a test in drush 8 too

Comment (finalhow) [reply templates/drupal-fix-permissions.sh.erb]: Doesn't work in drush 8...  :(

Comment (minorOffense) [reply templates/drupal-fix-permissions.sh.erb]: See if there's something equivalent

Comment (finalhow) [reply templates/drupal-fix-permissions.sh.erb]: Yes, I am looking for something like that.

Comment (finalhow) [reply templates/drupal-fix-permissions.sh.erb]: Closest I can get is: `drush core-status version` which outputs:

```
 Drupal version : 8.6.14 
 Drush version  : 9.5.2
```

**Pull request merged**